src/Type/WPObjectType.php

```diff
@@ -227,4 +232,17 @@ public function prepare_fields( $fields, $type_name, $config ) {
 		return $fields;
 	}
 
+	/**
+	 * Simple getter function so "$type_registry" can be used in hooks
+	 *
+	 * @param string $name  Member name.
+	 * @return TypeRegistry|null
+	 */
+	public function __get( $name ) {
+		if ( 'type_registry' === $name ) {
+			return $this->type_registry;
```

Comment:
I don't think this will ever get invoked. The magic method only gets called if there's not a matching method / property in the class, but you're returning a property with the same name you're checking against here. Unless the property is private, but in that case I would just change the property visibility instead of doing this.

Reply:
@kidunot89 added this to address some things he filters for WPGraphQL for WooCommerce. 

the property has changed to `public` so we can now get it without a method like this anyway.